Formula/k/kubevpn.rb

```diff
@@ -0,0 +1,36 @@
+class Kubevpn < Formula
+  desc "Offers a Cloud-Native Dev Environment that connects to your K8s cluster network"
+  homepage "https://www.kubevpn.cn"
+  url "https://github.com/kubenetworks/kubevpn.git",
+      tag:      "v2.2.11",
+      revision: "98d88ac542c06e8f66937e7f57b5ceae514b0c30"
+  license "MIT"
+  head "https://github.com/kubenetworks/kubevpn.git", branch: "master"
+
+  depends_on "go" => :build
+
+  def install
+    goos = Utils.safe_popen_read("#{Formula["go"].bin}/go", "env", "GOOS").chomp
+    goarch = Utils.safe_popen_read("#{Formula["go"].bin}/go", "env", "GOARCH").chomp
+    tags = "noassets"
+    project = "github.com/wencaiwulue/kubevpn/v2"
+    ldflags = %W[
+      -s -w
+      -X #{project}/pkg/config.Image=docker.io/naison/kubevpn:v#{version}
+      -X #{project}/pkg/config.Version=v#{version}
+      -X #{project}/pkg/config.GitCommit=#{Utils.git_head}
+      -X #{project}/cmd/kubevpn/cmds.BuildTime=#{time.iso8601}
+      -X #{project}/cmd/kubevpn/cmds.Branch=#{Utils.git_branch}
```

Comment:
There isn't a branch though, there is a tag.

Reply:
Yes, i know, it always `HEAD`, but it is ok